test/test_dependencies.jl

```diff
@@ -0,0 +1,47 @@
+using Mimi
+
+#list of URLs of branches of packages to test
+dependencies = ["https://github.com/davidanthoff/fund.jl/archive/master.zip"]
+
+#list of failed tests to build as you go
+errors = []
+
+#make a temporary directory to run the tests in
+run(`mkdir tmp_testing`)
+cd("tmp_testing")
+
+#loop through each dependent package
+for url in dependencies
+  #download the package
+  run(`wget $url`)
```

Comment:
This won't work on Windows. You can try to use [download](http://docs.julialang.org/en/release-0.4/stdlib/file/?highlight=download#Base.download) instead.